presto-bigquery/src/main/java/io/prestosql/plugin/bigquery/BigQueryClient.java

```diff
@@ -161,7 +161,7 @@ String selectSql(TableId table, String formattedColumns, String[] filters)
     {
         String tableName = fullTableName(table);
 
-        String whereClause = createWhereClause(filters)
+        String whereClause = createWhereClause()
```

Comment:
If `Optional.empty` always then can we remove whereClause ?

Reply:
OK, I will fix it up.